src/csvtsinputfileconn.cc

```diff
@@ -78,8 +83,9 @@ namespace dd
       return -1;
   }
 
-  int DDCsvTS::read_dir(const std::string &dir)
+  int DDCsvTS::read_dir(const std::string &dir, int test_id)
   {
+    (void)test_id;
```

Comment:
you are very right, this function read the dir passed in parameter _and_ the `_csv_test_fnames` _attribute_ , which is at very least extremely questionable :) 
this is inherited/copyied from csv connector behavior which does the same (attribute used is the one from the csv connector, as csvts connector inherits from csv connector). This could be refactored for clarity, or left as is because it works ...